content/en/_index.html

```diff
@@ -13,247 +13,1763 @@
         </div>
       </div>
-      <div class="margin-md-top-bottom is-align-items-center gap-md flex">
-        <div class="img-container-xsm">
-          <img src="sc_logo2.svg" alt="" />
+
+      <div class="sn-f-s">
+        <div class="feat-img">
+          <img src="sn2.png" alt="" class="in-feat-img" />
         </div>
-        <div class="content text-left">
-          <h4 class="is-5">DevSecOps</h4>
-          <p>For DevOps engineers</p>
+        <div class="feat-cont">
+          <h4 class="feat-cont-h">DevSecOps</h4>
+          <p class="feat-cont-d">For DevOps Engineer</p>
         </div>
       </div>
-      <div class="margin-md-top-bottom is-align-items-center gap-md flex">
-        <div class="img-container-xsm">
-          <img src="sc_logo3.svg" alt="" />
+
+      <div class="sn-f-s">
+        <div class="feat-img">
+          <img src="sn3.png" alt="" class="in-feat-img" />
         </div>
-        <div class="content text-left">
-          <h4 class="is-5">Auditing and Compliance</h4>
-          <p>For Security Operators</p>
+        <div class="feat-cont">
+          <h4 class="feat-cont-h">Auditing and Compliances</h4>
+          <p class="feat-cont-d">For Security Operator</p>
         </div>
       </div>
     </div>
-    <div class="img-long is-hidden-mobile">
-      <img src="notary_scenarios.svg" alt="" />
+    <div class="sn-img">
+      <img src="workflow1.png" alt="notary logo" class="sn-workflow" />
+    </div>
+  </div>
+</div>
+
+<!-- Contributed By Community -->
+<section class="contributors">
+  <p class="contributors-t">
+    Contributed by the community, in collaboration <br />with
+  </p>
+  <div
+    class="width-70 is-centered flex space-between is-align-items-center is-mobile-responsive text-centered-mobile"
+    id="contri-a"
+  >
+    <div class="b-box">
+      <div class="img-aws">
+        <img src="aws_logo.svg" alt="AWS logo" />
+      </div>
+    </div>
+    <div class="b-box">
+      <div class="img-cncf">
+        <img src="cncf_logo.svg" alt="CNCF logo" />
+      </div>
+    </div>
+    <div class="b-box">
+      <div class="img-doc">
+        <img src="docker_logo.svg" alt="Docker logo" />
+      </div>
+    </div>
+    <div class="b-box">
+      <div class="img-msft">
+        <img src="microsoft_logo.svg" alt="Microsoft logo" />
+      </div>
     </div>
   </div>
 </section>
 
 {{% blocks/lead color="white" %}}
-<section class="feature-container">
-  <div class="feature-text-container text-center">
-    <h1 class="title">Why the Notary Project is unique</h1>
-    <p class="width-60 is-centered font-size-md">
-      The Notary Project is aiming to provide enterprise-grade solutions and cross-industry standards for securing
-      software supply chain
+
+<div class="un-n">
+  <div class="u-cont">
+    <h4 class="u-cont-h">Why Notary is unique?</h4>
+    <p class="u-cont-d">
+      Notary is aiming to provide enterprise-grade solutions and cross-industry
```

Comment:
The Notary Project is aiming to provide enterprise-grade solutions and cross-industry standards for securing software supply chain